plugin/core/panels.py

```diff
@@ -163,17 +163,18 @@ def run(self, edit: sublime.Edit, window_id: int) -> None:
         to_process = WindowPanelListener.server_log_map.get(window_id) or []
         WindowPanelListener.server_log_map[window_id] = []
         with mutable(self.view):
+            new_lines = []
             for prefix, message in to_process:
                 message = message.replace("\r\n", "\n")  # normalize Windows eol
-                self.view.insert(edit, self.view.size(), "{}: {}\n".format(prefix, message))
+                new_lines.append("{}: {}\n".format(prefix, message))
```

Comment:
I've assumed that appending to list and concatenating later would be faster than appending to a string but that's not backed by any testing so I might be wrong.

Reply:
The `view.insert` also does inter-process serialization. So it makes sense that building the string here in python is faster.